lib/logstash/outputs/influxdb.rb

```diff
@@ -128,7 +128,7 @@ def receive(event)
     event_hash['name'] = event.sprintf(@series)
     sprintf_points = Hash[@data_points.map {|k,v| [event.sprintf(k), event.sprintf(v)]}]
     if sprintf_points.has_key?('time')
-      @logger.error("Cannot override value of time without 'allow_override_time'. Using event timestamp") unless @allow_override_time
+      @logger.error("Cannot override value of time without 'allow_time_override'. Using event timestamp") unless @allow_override_time
```

Comment:
Also this option is only used to print out a @logger.error message, which is quite strange.